BackEnd/config/db.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose')
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const MONGODB_URL = process.env.VITE_MONGODB_URL;
+
+const connectDB = async () => {
+    try {
+        const conn = await mongoose.connect(MONGODB_URL, {
+            useUnifiedTopology: true,
+            useNewUrlParser: true
+        })
+
+        console.log(`MongoDB connected: ${MONGODB_URL}`)
```

Comment:
It is not recommended to log the connection string to the console as the connection string contains user and password along with the endpoint to connect to your database. Incase someone gets access to your server logs he'll be able to see the connection string and easily access your database